synctest/run_bridge.go

```diff
@@ -0,0 +1,179 @@
+package synctest
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	logging "github.com/ipfs/go-log/v2"
+
+	"github.com/celestiaorg/celestia-node/logs"
+	"github.com/celestiaorg/celestia-node/node"
+	"github.com/celestiaorg/test-infra/testkit"
+	"github.com/celestiaorg/test-infra/testkit/appkit"
+	"github.com/celestiaorg/test-infra/testkit/nodekit"
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/testground/sdk-go/network"
+	"github.com/testground/sdk-go/run"
+	"github.com/testground/sdk-go/runtime"
+)
+
+func RunBridgeNode(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*2)
+	defer cancel()
+
+	os.Setenv("GOLOG_OUTPUT", "stdout")
+	level, err := logging.LevelFromString("INFO")
+	if err != nil {
+		return err
+	}
+	logs.SetAllLoggers(level)
```

Comment:
We already have a place with a general code path for nodes(syncNodes); better move common like this there.

Reply:
done